src/ray/protobuf/gcs_service.proto

```diff
@@ -504,6 +504,17 @@ message WaitPlacementGroupUntilReadyReply {
   GcsStatus status = 1;
 }
 
+message GetNamedPlacementGroupRequest {
+  // Name of the placement group.
+  string name = 2;
+}
+
+message GetNamedPlacementGroupReply {
+  GcsStatus status = 1;
```

Comment:
Do we need this status? What's this for?

Reply:
yeah, we should remove this.

Reply:
Yes, we need this, every rpc method reply need a status code, `GCS_RPC_SEND_REPLY` will set the status code for xxxReply